src/main/java/com/jalasoft/webservice/controller/DownloadController.java

```diff
@@ -27,13 +29,14 @@
 @RestController
 @RequestMapping("/download")
 public class DownloadController {
-
```

Comment:
insert space between the first line and the method

Reply:
Done